From a66be11c4a6b67237f098344bbbc0161af2aad4e Mon Sep 17 00:00:00 2001
From: Daan de Graaf <daandegraaf9@gmail.com>
Date: Sun, 22 Oct 2023 22:22:42 +0200
Subject: [PATCH] Add 'eudyptula' to the kernel version info.

Adding 'eudyptula' to the kernel version info by updating EXTRAVERSION, as
requested in the task.

Signed-of-by: Daan de Graaf <daandegraaf9@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5faaaf0fde6c..e8b6937b8f28 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 6
 SUBLEVEL = 0
-EXTRAVERSION = -rc6
+EXTRAVERSION = -eudyptula
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.34.1

